app/views/layouts/application.html.erb

```diff
@@ -29,6 +29,7 @@
       <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
 
+    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE"></script>
```

Comment:
Is there any other way to do this than to include this on every page? Since we don't need it for most pages, it will cause additional load time and possibly some slight JS load as well.